### ISSUE TITLE
Module not found: ESM Packages (since nanoevents introduced)

### ISSUE THREAD

**Issue by iamlinkus** [darkroomengineering/lenis#129]
Hey!

First of all, thanks a LOT for an amazing library and the fact that you've shared it with the open source community! It has been amazing working with Lenis. 

And now about my problem I'm facing. Today I started migrating my project code to the new `/app` dir (Next.js 13.2.1) and encountered a strange situation. Having not touched any dependencies in package.json, if I import a dependency (@studio-freight/lenis) into any component or layout or page, it throws:

```
./node_modules/@studio-freight/lenis/dist/lenis.js:1:6
Module not found: ESM packages (nanoevents) need to be imported. 
```

I tried downgrading Lenis to the version prior to nanoevents and test it out, and lo' and behold - it works! So I'm figuring that the problem lies with nanoevents.

**Comment by clementroche** [comment]
@iamlinkus did you check [this thread](https://github.com/vercel/next.js/issues/41865) ?

**Comment by iamlinkus** [comment]
@clementroche yep, none of the fixes work. Only downgrading lenis to the version prior to nanoevents fixes the issue.

**Comment by clementroche** [comment]
Since `/app` dir is still in beta, i'm not sure if we should investigate much more

**Comment by iamlinkus** [comment]
I totally agree with you. Just thought I'd share this in case the problem was with some old dependencies (If I'm not mistaken, maintainers of nanoevents have previously stated that it's not designed to work with react).

Either way, I think until `/app`dir gets to stability this shouldn't be a priority issue, since it works on the `/pages`dir just fine.

**Comment by clementroche** [comment]
Can you try with 1.0.1, it should have been fixed

**Comment by arzafran** [comment]
Closing because it should be fixed since v1.0.1
